src/app/components/registration/cognito-user-create.jsx

```diff
@@ -0,0 +1,160 @@
+import React, { useContext, useState } from 'react';
+import {
+  func,
+} from 'prop-types';
+import {
+  Button,
+  Paper,
+  ThemeProvider,
+  Typography,
+  makeStyles,
+} from '@material-ui/core';
+import { useFormik } from 'formik';
+import * as yup from 'yup';
+
+import theme from '../../themes/theme';
+import { ChplTextField } from '../util';
```

Comment:
We have an updated way of doing local (project specific) invites that removes the "relative directory" issue. This line can be replaced with:

```suggestion
import { ChplTextField } from 'components/util';
```

And if you take out the ThemeProvider I don't think you'll need the theme import either, but if you do:

```
import { theme } from 'themes';
```